app/helpers/activity_logger.py

```diff
@@ -1,9 +1,17 @@
 
 import datetime
 from flask_jwt_extended import get_jwt_identity
-from app.models import mongo
+from app.tasks import celery_app
+from flask_pymongo import MongoClient
+import os
+mongo = MongoClient(os.getenv('MONGO_URI', 'mongodb://localhost:27017/'))
```

Comment:
I think the MONGO_URI should be 'mongodb://localhost:27017/cranecloud' instead. Otherwise, it will always go to the exception and use the testing database.

Reply:
@Mubangizi

Reply:
Alright, I have adjusted it accordingly